x-pack/filebeat/input/httpjson/input_test.go

```diff
@@ -26,1167 +27,1250 @@ import (
 	"github.com/elastic/elastic-agent-libs/mapstr"
 )
 
-var testCases = []struct {
-	name         string
-	setupServer  func(testing.TB, http.HandlerFunc, map[string]interface{})
-	baseConfig   map[string]interface{}
-	handler      http.HandlerFunc
-	expected     []string
-	expectedFile string
+const dummyText = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
```

Comment:
Let's move this to below the test; it's not important what it is, so it doesn't need to be so promenant in the file.

Reply:
I have moved it below test cases.